test/interactions/pointerInteractionTests.ts

```diff
@@ -195,7 +195,7 @@ describe("Interactions", () => {
 
         assert.isFalse(enterCallback1Called, "callback 1 was disconnected from pointer enter interaction");
         assert.isTrue(enterCallback2Called, "callback 2 is still connected to the pointer enter interaction");
-
+        pointerInteraction.offPointerEnter(enterCallback2);
```

Comment:
A new interaction is created every single time in the `beforeEach` which exists to help the isolation